### DISCUSSION TITLE
xiaomi ax6使用这个源码编译，但是wifi默认设置有点问题

### DISCUSSION THREAD

**Discussion by rsd1987** [Boos4721/openwrt#4] [Q&A/答疑解惑] [answered]
AX6没有IOT部分的wifi，其他部分是一样的
但是默认设置部分会强制设置2.4G部分为5G
同时出现一个不正确的XIAOMI IOT的SSID
目前的解决办法只能删掉这个wifi配置重新设置
希望后面版本可以修复

**Comment by Boos4721** [comment, marked as answer]
本来就是给ax3600用的 
[ax6对着这个改](https://raw.githubusercontent.com/jarod360/openwrt/3b9af463e3339323f015267f31bbcbde46a80bf8/package/kernel/mac80211/files/lib/wifi/mac80211.sh)

**Reply by rsd1987** [reply]
这个已经改了，不过目前存在很多循环包依赖，可以简单处理下吗？

**Reply by Boos4721** [reply]
> 这个已经改了，不过目前存在很多循环包依赖，可以简单处理下吗？

图或者log

**Comment by jarod360** [comment]
老大看到源代码改了，但是2.4g和5g弄反了。 0是5g 1是2.4g

**Reply by Boos4721** [reply]
> 老大看到源代码改了，但是2.4g和5g弄反了。 0是5g 1是2.4g

只是单纯的同步了官方的mac80211改了下ssid加了个密码

**Reply by jarod360** [reply]
好的，可能ax6跟其他的是相反的！他的radio0 无线名称是Openwrt_5g

**Reply by Boos4721** [reply]
> 好的，可能ax6跟其他的是相反的！他的radio0 无线名称是Openwrt_5g

我之前的40xx系列是没问题的
